src/katsdpcontroller/generator.py

```diff
@@ -1190,17 +1277,36 @@ def get_baseline_index(a1, a2):
                 "{ports[aioconsole]}",
             ]
         )
-        output_config = {
-            "name": escape_format(stream.name),
-            "heap_accumulation_threshold": round(stream.int_time / heap_time),
-            "dst": f"{{endpoints_vector[multicast.{stream.name}_spead][{i}]}}",
-        }
-        xbgpu.command += [
-            "--corrprod",
-            ",".join(f"{key}={value}" for (key, value) in output_config.items()),
-        ]
+
+        for stream in streams:
+            if isinstance(stream, product_config.GpucbfBaselineCorrelationProductsStream):
+                output_config = {
+                    "name": escape_format(stream.name),
+                    "heap_accumulation_threshold": round(stream.int_time / heap_time),
+                    "dst": f"{{endpoints_vector[multicast.{stream.name}_spead][{i}]}}",
+                }
+                xbgpu.command += [
+                    "--corrprod",
+                    ",".join(f"{key}={value}" for (key, value) in output_config.items()),
+                ]
+            elif isinstance(stream, product_config.GpucbfTiedArrayChannelisedVoltageStream):
+                output_config = {
+                    "name": escape_format(stream.name),
+                    "dst": f"{{endpoints_vector[multicast.{stream.name}_spead][{i}]}}",
+                    "pol": stream.src_pol,
+                }
+                xbgpu.command += [
+                    "--beam",
+                    ",".join(f"{key}={value}" for (key, value) in output_config.items()),
+                ]
+
         if not configuration.options.develop.less_resources:
-            xbgpu.command += ["--src-affinity", "{cores[src]}", "--dst-affinity", "{cores[dst]}"]
+            xbgpu.command += [
+                "--src-affinity",
+                "{cores[src]}",
+                "--dst-affinity",
+                "{cores[dst]}",
+            ]
```

Comment:
Not sure why this changed - is it just `black` doing its thing?

Reply:
Yeah I think it's when I moved it under two for-loops. Rolled it back into one line and `black` is fine with it now.